vm/page_pool_alloc/src/lib.rs

```diff
@@ -36,7 +37,7 @@ pub mod save_restore {
     use vmcore::save_restore::SavedStateRoot;
 
     #[derive(Protobuf)]
-    #[mesh(package = "openhcl.pagepool")]
+    #[mesh(package = "openvmm.pagepool")]
```

Comment:
NOTE: I renamed this to reflect the move. Thoughts?

Reply:
the fact this used to be in openhcl, which has more strict saved-state compat requirements, is spooky.

might want to run this past @jstarks

Reply:
It's fine to rename because it's not supported in stable saved state anywhere, but do we want to prefix it with something else that's _not_ `openvmm`?